pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "metricflow"
 version = "0.205.0"
 description = "Translates a simple metric definition into reusable SQL and executes it against the SQL engine of your choice."
 readme = "README.md"
-requires-python = ">=3.8,<3.12"
+requires-python = ">=3.9,<=3.12"
```

Comment:
Python 3.8 is supported through October, so we need to to support 3.8 through the dbt-core release following official EOL.